htsohm/PseudoMaterial.py

```diff
@@ -0,0 +1,12 @@
+class PseudoMaterial:
+
+    def __init__(self, uuid):
+        self.uuid = uuid
```

Comment:
For classes, all instance vars (i.e. self.*) should be defined in the __init__ method. You can just use named arguments with defaults for the optional parameters so they're aren't required, but you'll want a list of self.lattice_constants, self.atom_types, etc here so you know what is expected. Right now your __repr__ fails because it is expecting instance variables to exist on the object which don't.